torchio/transforms/augmentation/intensity/random_gamma.py

```diff
@@ -0,0 +1,56 @@
+from typing import Tuple, Optional, List
+import torch
+from ....torchio import DATA, TypeRangeFloat
+from ....data.subject import Subject
+from .. import RandomTransform
+
+
+class RandomGamma(RandomTransform):
+    r"""Change contrast of an image by setting its values to the power
```

Comment:
I'm not sure "set a value to the power x" makes sense to me. What about "raising" instead of "setting"?